pkg/api/server/register_generate.go

```diff
@@ -8,6 +8,68 @@ import (
 )
 
 func (s *APIServer) registerGenerateHandlers(r *mux.Router) error {
+	// swagger:operation GET /libpod/generate/{name:.*}/systemd libpod libpodGenerateSystemd
+	// ---
+	// tags:
+	//  - containers
+	//  - pods
+	// summary: Generate Systemd Units.
+	// description: Generate Systemd Units based on a pod or container.
+	// parameters:
+	//  - in: path
+	//    name: name:.*
+	//    type: string
+	//    required: true
+	//    description: Name or ID of the container or pod.
+	//  - in: query
+	//    name: name
```

Comment:
Having name as both a path and query parameter in one endpoint seems a point of future confusion. I understand why this was done, but would this be better as `useName`?

Reply:
`useName` sounds good :+1: